### ISSUE TITLE
Poor functional BOLD brain masks

### ISSUE THREAD

**Issue by dlevitas** [nipreps/fmriprep#2836]
### What happened?

I've noticed that with a subset of my dataset, the brain masks are poorly generated. I've attached two screenshots that demonstrate the issue. I believe this is tied to #2731 , which appears to be a lingering issue. 

I can try to re-run fmriprep with new working directories and not re-use the freesurfer outputs, to see if that alleviates the issue. Additionally, I see that fmriprep version 22.0.0 is released, so I can try using that version as well. I'll report back once those are finished.

### What command did you use?

```shell
                unset PYTHONPATH; singularity run \
		-B ${bids_root_dir}:${bids_root_dir} \
		-B /N/dcwan/projects/irf/templateflow:/opt/templateflow
		-B ${work_dir}:${work_dir} \
		-B ${output_dir}:${output_dir} \
		/N/dcwan/projects/irf/containers/fmriprep-${fmriprep_version}.simg 			\
		$bids_root_dir $output_dir													\
		participant																\
		--participant-label $s 													\
		--skip-bids-validation 													\
		--md-only-boilerplate													\
		--fs-license-file $FREESURFER_HOME/license.txt							\
		--fs-subjects-dir $output_dir/sourcedata/freesurfer/sub-${s}			\
		--output-spaces MNI152NLin6Asym:res-2									\
		--nthreads $cores														\
		--stop-on-first-crash 													\
		--resource-monitor 														\
		--low-mem 																\
		--mem_mb $reduced_mem 													\
		--use-plugin $bids_root_dir/fmriprep_plugin_${s}.yml 					\
		--use-aroma 															\
		--aroma-melodic-dimensionality 100 										\
		--dummy-scans $dummy_scans 												\
		--verbose 																\
		--bids-filter-file ${bids_root_dir}/code/bids_filter_file.json 			\
		--output-layout bids 													\
		-w $work_dir/sub-${s}"
```

### What version of fMRIPrep are you running?

21.0.2

### How are you running fMRIPrep?

Singularity

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

FreeSurfer, Work directory

### Please copy and paste any relevant log output.

```shell
N/A
```

### Additional information / screenshots

![bad_mask1](https://user-images.githubusercontent.com/25011343/186305086-78d9c5d4-3b15-4ff3-abf0-5ec910ff708b.png)
![bad_mask2](https://user-images.githubusercontent.com/25011343/186305092-795fd2f2-6c0d-41fc-a3ec-419d52002ea0.png)

**Comment by dlevitas** [comment]
I re-ran my dataset with version 22.0.0, using clean freesurfer and working directories. Unfortunately the brain mask issue persists.
![sub-02_task-pacpal_run-1_desc-bbregister_bold](https://user-images.githubusercontent.com/25011343/186511197-f85a6f0b-bd6b-493f-a50d-264938aa93a2.svg)

**Comment by dlevitas** [comment]
Using the `--use-syn-sdc` did not solve my issue, but I was able to determine what has caused my problem. 

For context, I collected a SBRef for each functional BOLD image. Additionally, I collected both magnitude AND phase, as I'm hoping to use the phase data for a secondary analysis. For whatever reason, my Siemens Prisma scanner output a SBRef for the magnitude and the phase (thus 2 SBRef for each functional BOLD image). I thought I was excluding any phase images from being used by fmriprep by including that information in my `--bids-filter-file bids_filter_file.json`; however, this doesn't appear to be the case, as fmriprep was using the SBRef from the phase data (`part-phase_sbref`) as the reference for the BOLD mask. My solution was to simply remove the SBRef phase data from my BIDS folder. 

I'll therefore close out this issue, but open a new one regarding the issue with the `bids-filter-file`
